cmd/main.go

```diff
@@ -4,7 +4,7 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/spiceai/gospice/v4"
+	gospice "github.com/spiceai/gospice/v4.0.1"
```

Comment:
Go modules don't include the minor versions in their import path - that is handled by a git tag.
```suggestion
	gospice "github.com/spiceai/gospice/v4"
```

Reply:
@phillipleblanc  - noted. Thank you for review. Let me review best practices and how tagging works for golang and I'll respond